components/load_from_files/tests/test_load_from_files_component.py

```diff
@@ -0,0 +1,295 @@
+import gzip
```

Comment:
Nice that you have tests. They are however not ran in the current CI/CD pipeline since we only call pytest on the `fondant/tests/` folder. This is something you could solve in your other PR. 

I'm assuming you ran these manually

Reply:
Yes , I ran it manually and @RobbeSneyders asked me to create tests within the components itself so I created within that same folder